devhelp/src/dhp-settings.c

```diff
@@ -0,0 +1,367 @@
+/* Included directly into dhp-object.c just to keep this whole mess out
```

Comment:
What is this comment about?

Reply:
It was already in this file.
It was Matthew who had put.